python/cudf/cudf/core/column/string.py

```diff
@@ -730,42 +730,57 @@ def wrap(self, width, **kwargs):
             msg = f"width must be of integer type, not {type(width).__name__}"
             raise TypeError(msg)
 
-        expand_tabs = kwargs.get("expand_tabs", False)
+        expand_tabs = kwargs.get("expand_tabs", None)
         if expand_tabs:
             raise NotImplementedError("`expand_tabs=True` is not supported")
-        warnings.warn(
-            "wrap current implementation defaults to `expand_tabs`=False"
-        )
+        elif expand_tabs is None:
+            warnings.warn(
+                "wrap current implementation defaults to `expand_tabs`=False"
+            )
 
-        replace_whitespace = kwargs.get("replace_whitespace", True)
+        replace_whitespace = kwargs.get("replace_whitespace", None)
         if not replace_whitespace:
             raise NotImplementedError(
                 "`replace_whitespace=False` is not supported"
             )
+        elif replace_whitespace is None:
+            warnings.warn(
+                "wrap current implementation defaults to \
+                    `replace_whitespace`=True"
+            )
```

Comment:
We already have the correct default here:
```suggestion
        replace_whitespace = kwargs.get("replace_whitespace", True)
        if not replace_whitespace:
            raise NotImplementedError(
                "`replace_whitespace=False` is not supported"
            )
```